apps/frontend/app/Header/NavBar.tsx

```diff
@@ -2,12 +2,14 @@
 
 import { useState } from "react";
 import Image, { StaticImageData } from "next/image";
-import Icon from "./Icon"; 
 import SubMenu from "./SubMenu";
+import { ReactNode } from "react";
+import RutasNavegacion from "../../ routes"; 
```

Comment:
This import could be improved... make it cleaner.

Reply:
change the name of the route to make it clearer